src/gateway/error.rs

```diff
@@ -55,7 +55,7 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::BuildingUrl => f.write_str("Error building url"),
+            Self::BuildingUrl => f.write_str("Error: Failed to build url"),
```

Comment:
none of the other variants use the `Error:` prefix. All together i'd prefer you didn't touch the code at all in a PR that "fixes" the README.